app/webpack/stylesheets/components/_card.scss

```diff
@@ -1,6 +1,7 @@
 .app-card--defendant {
+  width: 445px;
```

Comment:
This hard coding of width bothers me somewhat as when your scaling the page for different screen sizes it doesnt help. If its just the word wrapping thats the problem could we use `overflow-wrap: inherit;` to potentially fix this? The other thing is the Key width is 30% on the card `.govuk-summary-list__key` so we could potentially override this to be 35% if you wanted to.